scripts/skills/effects/rf_sprint_effect.nut

```diff
@@ -17,34 +19,37 @@ this.rf_sprint_effect <- ::inherit("scripts/skills/skill", {
 	{
 		local tooltip = this.skill.getTooltip();
 
-		tooltip.extend([
-			{
-				id = 10,
-				type = "text",
-				icon = "ui/icons/action_points.png",
-				text = ::MSU.Text.colorGreen(-1) + " Action Points spent per tile traveled"
-			},
-			{
-				id = 10,
+		tooltip.push({
+			id = 10,
+			type = "text",
+			icon = "ui/icons/action_points.png",
+			text = ::MSU.Text.colorGreen(-1) + " Action Points spent per tile traveled"
+		});
+
+		if (this.m.FatigueMult != 1.0)
+		{
+			tooltip.push({
+				id = 11,
 				type = "text",
 				icon = "ui/icons/fatigue.png",
-				text = ::MSU.Text.colorRed("100%") + " more Fatigue built per tile traveled"
-			}
-			{
-				id = 10,
-				type = "text",
-				icon = "ui/icons/warning.png",
-				text = ::MSU.Text.colorRed("Will expire upon using any skill")
-			}
-		]);
+				text = ::MSU.Text.colorizeMult(this.m.FatigueMult, {InvertColor = true, AddSign = true}) + " Fatigue built per tile traveled"
+			});
+		}
+
+		tooltip.push({
+			id = 12,
+			type = "text",
+			icon = "ui/icons/warning.png",
+			text = ::MSU.Text.colorRed("Will expire upon using any skill")
```

Comment:
I will wait with this PR until we find a common ground for how to handle these warnings.
I already pointed out the redundant double warning (warning icon + red text color) to midas and now realised that it snuck into my commit aswell.